Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <LangVersion>7.2</LangVersion>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
Had to disable warnings about missing XML docs (1591).

Reply:
There are two or three warnings that should be added here in total. Check the docs for SA0001 for the second, which applies to parameters. The third applies to type parameters by I've only seen it needed rarely.

Reply:
Thanks. Seems there are [two to add](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA0001.md), namely CS1573 and CS1591. I'll update that in this file.